doc/tools/docgram/common.edit_mlg

```diff
@@ -1097,13 +1090,11 @@ simple_tactic: [
 | WITH "subst" OPT ( LIST1 var )
 | DELETE "subst"
 | DELETE "congruence"
-| DELETE "congruence" int
+| DELETE "congruence" integer
 | DELETE "congruence" "with" LIST1 constr
-| REPLACE "congruence" int "with" LIST1 constr
-| WITH "congruence" OPT int OPT ( "with" LIST1 constr )
+| REPLACE "congruence" integer "with" LIST1 constr
+| WITH "congruence" OPT integer OPT ( "with" LIST1 constr )
```

Comment:
Does an `integer` really make sense here (as well as in `g_congruence.mlg`? The doc of `congruence` in the tactics chapter says `natural`.